sdk/eventgrid/azure-eventgrid/azure/eventgrid/aio/_operations/_patch.py

```diff
@@ -95,16 +191,26 @@ async def publish_cloud_events(
         :rtype: None
         :raises ~azure.core.exceptions.HttpResponseError:
         """
+        
+        # Check that the body is a CloudEvent or list of CloudEvents even if dict
+        if isinstance(body, dict) or (isinstance(body, list) and isinstance(body[0], dict)):
```

Comment:
in the case of body[0] being a dict and then body[1] not being a dict?

Reply:
For right now would be unsupported in v2 -- is also not supported in EGv1

Reply:
I was thinking if this section should just be there for dicts and lists.  If someone sends in body[0] as a list and body[1] as dict they'd fall in to the catch all.